great_expectations/expectations/expectation.py

```diff
@@ -802,7 +801,7 @@ def get_domain_kwargs(
 
     def get_success_kwargs(
         self, configuration: Optional[ExpectationConfiguration] = None
-    ):
+    ) -> Dict[str, Any]:
```

Comment:
@billdirks Nice!  Now, since you did this -- you may want to update the total count (please confirm with @cdkini).  Thanks!

Reply:
Done.